pypeman/tests/test_channel.py

```diff
@@ -74,6 +86,82 @@ def test_no_node_base_channel(self):
         self.start_channels()
         self.loop.run_until_complete(chan.handle(msg))
 
+    def test_done_callback(self):
+        """ Whether BaseChannel done_callback is working """
```

Comment:
you should also checked that it is not called in the cases where it should not be called.